hypervisor/arch/x86/guest/vcpu.c

```diff
@@ -206,6 +206,12 @@ void set_vcpu_regs(struct vcpu *vcpu, struct acrn_vcpu_regs *vcpu_regs)
 
 	ectx->ldtr.selector = vcpu_regs->ldt_sel;
 	ectx->tr.selector = vcpu_regs->tr_sel;
+	ectx->ldtr.base = 0UL;
```

Comment:
This is to set default LDTR/GDTR/TR registers... Add a comment for now to remind us?
It may not be correct in future if the API is called not for reset purpose.

Reply:
Sure.